Cargo.lock

```diff
@@ -3515,10 +3516,14 @@ dependencies = [
 name = "fuel-core-metrics"
 version = "0.39.0"
 dependencies = [
+ "derive_more",
```

Comment:
err, shouldn't these have been in master already?

Reply:
Everything looks right, it is dependencies of the `fuel-core-metrics` crate and we added a new crate here=)